### PR TITLE
Cut 0.19.0 release

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Minikube Release Notes
 
+## Version 0.19.0 - 5/3/2017
+* Updated nginx ingress to v0.9-beta.3
+* Updated kube-dns to 1.14.1
+* Added optional `--name` flag to all `minikube` commands to support multiple minikube instances
+* Increased localkube boot speed by removing dependancy on the network being up
```
Comment (r2d4) [inline CHANGELOG.md]: nit: dependency

**Comment by aaron-prindle** [comment]
@minikube-bot retest this please

**Comment by aaron-prindle** [comment]
@minikube-bot retest this please

**Comment by aaron-prindle** [comment]
@minikube-bot retest this please

**Comment by aaron-prindle** [comment]
@minikube-bot retest this please

**Comment by aaron-prindle** [comment]
@minikube-bot retest this please

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Minikube Release Notes
 
+## Version 0.19.0 - 5/3/2017
+* Updated nginx ingress to v0.9-beta.3
```
Comment (secat) [inline CHANGELOG.md]: the new version should be `v0.9.0-beta.4`

**Comment by aaron-prindle** [comment]
OSX tests appear to be failing consistently for this.  Seeing:
```
	util.go:66: Error running command: start --vm-driver=virtualbox --cpus=4 --v=100  --docker-env=FOO=BAR --docker-env=BAZ=BAT --docker-opt=debug --docker-opt=icc=true E0508 10:31:34.301068   56385 start.go:119] Error starting host: Error loading existing host. Please try running [minikube delete], then run [minikube start] again.: open /Users/jenkins/.minikube/machines/minikube/config.json: no such file or directory.
		
		 Retrying.
		E0508 10:31:34.301404   56385 start.go:125] Error starting host:  Error loading existing host. Please try running [minikube delete], then run [minikube start] again.: open /Users/jenkins/.minikube/machines/minikube/config.json: no such file or directory
```

**Comment by aaron-prindle** [comment]
@minikube-bot retest this please

**Comment by aaron-prindle** [comment]
@minikube-bot retest this please

**Pull request merged**